src/rainflow.py

```diff
@@ -122,19 +124,26 @@ def extract_cycles(series, left=False, right=False):
             points.popleft()
 
 
-def count_cycles(series, ndigits=None, left=False, right=False):
+def count_cycles(series, ndigits=None, nbins=None, binsize=None, left=False, right=False):
     """Count cycles in the series.
 
     Parameters
     ----------
     series : iterable sequence of numbers
     ndigits : int, optional
         Round cycle magnitudes to the given number of digits before counting.
+    nbins : int, optional
+        Specifies the number of cycle-counting bins
+    binsize : int, optional
```

Comment:
binsize should be of type float
```suggestion
    binsize : float, optional
```

Reply:
ok